StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1141UseTupleSyntax.cs

```diff
@@ -227,7 +233,9 @@ private static void CheckGenericName(SyntaxNodeAnalysisContext context, GenericN
         private static bool IsValueTuple(SyntaxNodeAnalysisContext context, ExpressionSyntax syntax)
         {
             var symbolInfo = context.SemanticModel.GetSymbolInfo(syntax, context.CancellationToken);
-            return (symbolInfo.Symbol is ITypeSymbol typeSymbol) && typeSymbol.IsTupleType();
+            return symbolInfo.Symbol is INamedTypeSymbol typeSymbol
+                && typeSymbol.IsTupleType()
+                && typeSymbol.TupleElements().Length > 1;
```

Comment:
I would prefer that you rename the function to indicate that it is no longer considering tuples with 1 element.

Reply:
Now fixed